diff --git a/main.go b/main.go
index d7ce1db..91137d5 100644
--- a/main.go
+++ b/main.go
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"os"
 	"github.com/jesseduffield/lazygit/pkg/app"
 )
 
@@ -13,6 +14,7 @@ var (
 )
 
 func main() {
+	os.Setenv("__ZOS_QUICKEXIT", "1")
 	ldFlagsBuildInfo := &app.BuildInfo{
 		Commit:      commit,
 		Date:        date,
diff --git a/pkg/config/user_config.go b/pkg/config/user_config.go
index 192d138..ad704fa 100644
--- a/pkg/config/user_config.go
+++ b/pkg/config/user_config.go
@@ -770,7 +770,7 @@ func GetDefaultConfig() *UserConfig {
 			EnlargedSideViewLocation: "left",
 			WrapLinesInStagingView:   true,
 			UseHunkModeInStagingView: true,
-			Language:                 "auto",
+			Language:                 "en",
 			TimeFormat:               "02 Jan 06",
 			ShortTimeFormat:          time.Kitchen,
 			Theme: ThemeConfig{
diff --git a/pkg/i18n/i18n.go b/pkg/i18n/i18n.go
index 55c2308..9a8d807 100644
--- a/pkg/i18n/i18n.go
+++ b/pkg/i18n/i18n.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 
 	"dario.cat/mergo"
-	"github.com/cloudfoundry/jibber_jabber"
+//	"github.com/cloudfoundry/jibber_jabber"
 	"github.com/go-errors/errors"
 	"github.com/samber/lo"
 	"github.com/sirupsen/logrus"
@@ -20,8 +20,8 @@ func NewTranslationSetFromConfig(log *logrus.Entry, configLanguage string) (*Tra
 	if err != nil {
 		return nil, err
 	}
-
-	if configLanguage == "auto" {
+        configLanguage = "en"
+/*	if configLanguage == "auto" {
 		language := detectLanguage(jibber_jabber.DetectIETF)
 		for _, languageCode := range languageCodes {
 			if strings.HasPrefix(language, languageCode) {
@@ -32,7 +32,7 @@ func NewTranslationSetFromConfig(log *logrus.Entry, configLanguage string) (*Tra
 		// Detecting a language that we don't have a translation for is not an
 		// error, we'll just use English.
 		return EnglishTranslationSet(), nil
-	}
+	}*/
 
 	if configLanguage == "en" {
 		return EnglishTranslationSet(), nil
