diff --git a/pkg/config/user_config.go b/pkg/config/user_config.go
index 9d5fb3742..530a7f180 100644
--- a/pkg/config/user_config.go
+++ b/pkg/config/user_config.go
@@ -702,7 +702,7 @@ func GetDefaultConfig() *UserConfig {
 			MainPanelSplitMode:       "flexible",
 			EnlargedSideViewLocation: "left",
 			WrapLinesInStagingView:   true,
-			Language:                 "auto",
+			Language:                 "en",
 			TimeFormat:               "02 Jan 06",
 			ShortTimeFormat:          time.Kitchen,
 			Theme: ThemeConfig{
diff --git a/pkg/i18n/i18n.go b/pkg/i18n/i18n.go
index 3848d42cf..a1cc50862 100644
--- a/pkg/i18n/i18n.go
+++ b/pkg/i18n/i18n.go
@@ -7,7 +7,7 @@ import (
 	"io/fs"
 	"strings"
 
-	"github.com/cloudfoundry/jibber_jabber"
+//	"github.com/cloudfoundry/jibber_jabber"
 	"github.com/go-errors/errors"
 	"github.com/imdario/mergo"
 	"github.com/samber/lo"
@@ -19,8 +19,8 @@ func NewTranslationSetFromConfig(log *logrus.Entry, configLanguage string) (*Tra
 	if err != nil {
 		return nil, err
 	}
-
-	if configLanguage == "auto" {
+        configLanguage = "en"
+/*	if configLanguage == "auto" {
 		language := detectLanguage(jibber_jabber.DetectIETF)
 		for _, languageCode := range languageCodes {
 			if strings.HasPrefix(language, languageCode) {
@@ -31,7 +31,7 @@ func NewTranslationSetFromConfig(log *logrus.Entry, configLanguage string) (*Tra
 		// Detecting a language that we don't have a translation for is not an
 		// error, we'll just use English.
 		return EnglishTranslationSet(), nil
-	}
+	}*/
 
 	if configLanguage == "en" {
 		return EnglishTranslationSet(), nil
